test/predictions/repo_test.exs

```diff
@@ -58,6 +58,102 @@ defmodule Predictions.RepoTest do
       end
     end
 
+    test "filtes out predictions with no departure" do
+      five_minutes_in_future = DateTime.add(Timex.now(), 5, :minute)
+
+      five_minutes_in_future_string =
+        Timex.format!(five_minutes_in_future, "{ISO:Extended:Z}")
+
+      expect(MBTA.Api.Mock, :get_json, fn _, _ ->
+        %JsonApi{
+          data: [
+            %JsonApi.Item{
+              attributes: %{
+                "departure_time" => nil,
+                "arrival_time" => five_minutes_in_future_string,
+                "direction_id" => 1
+              },
+              relationships: %{
+                "route" => [
+                  %{
+                    id: "Red"
+                  }
+                ],
+                "trip" => [],
+                "vehicle" => [],
+                "stop" => [
+                  %{id: "StopID"}
+                ]
+              }
+            },
+            %JsonApi.Item{
+              attributes: %{
+                "departure_time" => five_minutes_in_future_string,
+                "arrival_time" => five_minutes_in_future_string,
+                "direction_id" => 1
+              },
+              relationships: %{
+                "route" => [
+                  %{
+                    id: "Red"
+                  }
+                ],
+                "trip" => [],
+                "vehicle" => [],
+                "stop" => [
+                  %{id: "StopID"}
+                ]
+              }
+            }
+          ]
+        }
+      end)
+
+      # Route for calculating display time
+      expect(MBTA.Api.Mock, :get_json, fn _, _ ->
+        %JsonApi{
+          data: [
+            %JsonApi.Item{
+              id: "Red",
+              attributes: %{
+                "type" => "1",
+                "long_name" => "Red Test Subway",
+                "direction_names" => ["North Test Name", "South Test Name"],
+                "direction_destinations" => ["North Test", "South Test"]
```

Comment:
These are all magic strings. As per the testing doc we all reviewed and agreed to, we should be using Faker for these (and all others in this file).

https://www.notion.so/mbta-downtown-crossing/The-Four-Legs-of-Quality-Programming-b35248799da84e02b93887361d5bf343